pinot-core/src/test/java/org/apache/pinot/core/transport/QueryRoutingTest.java

```diff
@@ -211,7 +211,7 @@ public void testServerDown()
     assertEquals(serverResponse.getResponseSize(), 0);
     assertEquals(serverResponse.getDeserializationTimeMs(), 0);
     // Query should early terminate
-    assertTrue(System.currentTimeMillis() - startTimeMs < 1000);
+    assertTrue(System.currentTimeMillis() - startTimeMs < 1010);
```

Comment:
Seems unrelated. If we find this test is flaky, maybe having another PR to fix it?

Reply:
Will put it in a separate PR. Test case fails every once a while because it needs a little extra time over 1000 ms timeout to validate the timeout.

Reply:
See flaky test PR at https://github.com/apache/incubator-pinot/pull/6893. I will revert the change from here once the flaky test PR is merged.